Dockerfile

```diff
@@ -1,22 +1,25 @@
 # This file is part of REANA.
-# Copyright (C) 2017, 2018, 2020 CERN.
+# Copyright (C) 2017, 2018, 2020, 2021 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-FROM ubuntu:16.04
+FROM rabbitmq:3.8.9-management
+
 # hadolint ignore=DL3009
-RUN apt-get update
-# hadolint ignore=DL3008
-RUN apt-get -y install --no-install-recommends rabbitmq-server
+RUN apt-get update && apt-get install -y --no-install-recommends curl=7.58.0-2ubuntu3.13 ca-certificates=20210119~18.04.1
 RUN apt-get -y autoremove && apt-get -y clean
+
 ENV RABBITMQ_NODENAME=rabbit@localhost
 ARG DEBUG=0
-RUN if [ "${DEBUG}" -gt 0 ]; then rabbitmq-plugins enable rabbitmq_management; fi;
-# hadolint ignore=DL3001
-RUN service rabbitmq-server start
+
+RUN curl -L "https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/releases/download/3.8.9/rabbitmq_delayed_message_exchange-3.8.9-0199d11c.ez" > "$RABBITMQ_HOME/plugins/rabbitmq_delayed_message_exchange-3.8.9-0199d11c.ez"
+RUN chown rabbitmq:rabbitmq "$RABBITMQ_HOME/plugins/rabbitmq_delayed_message_exchange-3.8.9-0199d11c.ez"
+
+RUN rabbitmq-plugins enable --offline rabbitmq_delayed_message_exchange
+RUN rabbitmq-plugins enable --offline rabbitmq_consistent_hash_exchange
+RUN if [ "${DEBUG}" -lt 1 ]; then rabbitmq-plugins disable --offline rabbitmq_management; fi;
```

Comment:
Trailing ";" after "fi" should not be necessary. (But I see we have it everywhere...)